qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -184,6 +186,26 @@ def __init__(self,
         """
         super().__init__(name, address, terminator=terminator, **kwargs)
 
+        idn = self.IDN.get()
+
+        self.has_firmware_a_02_10_or_above = LooseVersion(idn["firmware"]) >=\
+                                             LooseVersion("A.02.10")
+
+        self.has_option_001 = '001' in self._options()
```

Comment:
what are the possible outputs of the options method? is it one of `"", "001", "002"` or else? or are combinations possible?

i just wonder whether we should define these "001"s are contants as opposed to describing them in plain text in the docstring of _options method

Reply:
combinations are possible, options method returns a tuple of installed options

Reply:
ok, then it's good as is :)